.travis.yml

```diff
@@ -0,0 +1,20 @@
+# Travis CI (MIT License) configuration file for PubLibData application
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Declare versions of PHP to use. Use one decimal max.
+php:
+  # aliased to a recent 5.5.x version
+  - 7.0
+  - 5.6
+
+before_script:
+  - composer install
+
+script:
+  # Search for PHP syntax errors.
+  - find . \( -name '*.php' \) -exec php -lf {} \;
+  # PHPUnit for testing
```

Comment:
I was wondering about PHPCS too. For the theme development I do, I usually run a syntax checker like this, and then also a code analysis tool for conformance with code standards. Hearing you think the same way, I'll add it to this PR.